src/applications/appeals/996/tests/validations/issues.unit.spec.js

```diff
@@ -2,7 +2,7 @@ import { expect } from 'chai';
 import sinon from 'sinon';
 
 import { getDate } from '../../utils/dates';
-import { SELECTED, MAX_LENGTH } from '../../constants';
+import { SELECTED, MAX_LENGTH } from '../../../shared/constants';
```

Comment:
This `MAX_LENGTH` needs both `SELECTIONS` from shared constants and `ISSUE_NAME` from 996 constants